packages/paste-website/src/components/site-wrapper/index.tsx

```diff
@@ -64,6 +67,12 @@ const SiteWrapper: React.FC<SiteWrapperProps> = ({pathname, children}) => {
     <ActiveSiteThemeProvider>
       <SiteThemeProvider>
         <NavigationContext.Provider value={{...navigationQueryData, pathname}}>
+          <SkipLinkContainer>
+            <Stack orientation="horizontal" spacing="space60">
+              <Anchor href="#content">Skip to content</Anchor>
+              <Anchor href="#search-input">Skip to search</Anchor>
```

Comment:
Might be worth moving this to a constant file too since it is now being referenced in a second location. This makes the code less brittle, if for some reason in the future someone were to update search and change this ID.

Reply:
Changes have been pushed.